src/templates/project.js

```diff
@@ -5,54 +5,158 @@ import Helmet from 'react-helmet';
 import Content, { HTMLContent } from '../components/Content';
+import Img from 'gatsby-image';
 
 export const ProjectTemplate = ({
-  content,
-  contentComponent,
   description,
-  tags,
   title,
-  helmet
+  helmet,
+  title1,
+  description1,
+  title2,
+  description2,
+  mainimage,
+  miniatureimage1,
+  miniatureimage2,
+  relatedproducts
 }) => {
-  const PostContent = contentComponent || Content;
-
   return (
-    <section className="section">
+    <section>
       {helmet || ''}
-      <div className="container content">
-        <div className="columns">
-          <div className="column is-10 is-offset-1">
+      <div className="">
+        <div>
+          <div className="project-title-section">
             <h1 className="title is-size-2 has-text-weight-bold is-bold-light">
               {title}
             </h1>
-            <p>{description}</p>
-            <PostContent content={content} />
-            {tags && tags.length ? (
-              <div style={{ marginTop: `4rem` }}>
-                <h4>Tags</h4>
-                <ul className="taglist">
-                  {tags.map(tag => (
-                    <li key={tag + `tag`}>
-                      <Link to={`/tags/${kebabCase(tag)}/`}>{tag}</Link>
-                    </li>
-                  ))}
-                </ul>
+            <p className="is-size-5">{description}</p>
+          </div>
+          <div>
+            {mainimage ? (
+              <div>
+                {typeof mainimage === 'string' ? (
+                  <img
+                    src={mainimage}
+                    style={{ maxHeight: '720px', width: '100%' }}
+                  />
+                ) : (
+                  <Img
+                    fluid={mainimage.childImageSharp.fluid}
+                    alt={`featured image thumbnail for project ${title1}`}
+                    style={{ maxHeight: '720px' }}
+                  />
+                )}
               </div>
             ) : null}
           </div>
+          <div className="project-info-section">
+            <div className="project-info-first">
+              <h1 className="title is-size-2 has-text-weight-bold is-bold-light">
+                {title1}
+              </h1>
+              <p>{description1}</p>
+              <div className="columns project-miniature-images">
+                {miniatureimage1 ? (
+                  <div className="column is-two-fifths">
+                    {typeof miniatureimage1 === 'string' ? (
+                      <img src={miniatureimage1} style={{ width: '40%' }} />
+                    ) : (
+                      <Img
+                        fluid={miniatureimage1.childImageSharp.fluid}
+                        alt={`featured image thumbnail for project ${title1}`}
+                        style={{ maxHeight: '270px', maxWidth: '380px' }}
+                      />
+                    )}
+                  </div>
+                ) : null}
+                {miniatureimage2 ? (
```

Comment:
Should have onClick link for the whole picture, not just the title of the product?

Reply:
Consider it done